docs/guides/07-zoom-based-styles.md

```diff
@@ -1,4 +1,4 @@
-## Zoom-based Styling
+## Zoom-based styles
```

Comment:
I don't understand the purpose of this change. `Zoom-based styles` sounds to me like if there were a fixed number of zoom-based styles.